base/src/main/java/com/smartdevicelink/managers/ManagerUtility.java

```diff
@@ -0,0 +1,86 @@
+package com.smartdevicelink.managers;
+
+import com.smartdevicelink.proxy.rpc.ImageField;
+import com.smartdevicelink.proxy.rpc.TextField;
+import com.smartdevicelink.proxy.rpc.WindowCapability;
+import com.smartdevicelink.proxy.rpc.enums.ImageFieldName;
+import com.smartdevicelink.proxy.rpc.enums.TextFieldName;
+
+/**
+ * <strong>ManagerUtility</strong> <br>
+ * <p>
+ * Static Methods to be used throughout the Manager classes <br>
+ */
+public class ManagerUtility {
+
+    public static class WindowCapabilityUtility {
+
+        /**
+         * Check to see if WindowCapability has an ImageFieldName of a given name.
+         *
+         * @param windowCapability WindowCapability representing the capabilities of the desired window
+         * @param name ImageFieldName representing a name of a given Image field that would be stored in WindowCapability
+         * @return true if name exist in WindowCapability else false
+         */
+        public static boolean hasImageFieldOfName(WindowCapability windowCapability, ImageFieldName name) {
+            if (windowCapability == null) {
+                return false;
+            }
+            if (windowCapability.getImageFields() != null) {
+                for (ImageField field : windowCapability.getImageFields()) {
+                    if (field != null && field.getName() != null && field.getName().equals(name)) {
```

Comment:
```suggestion
                    if (field != null && name.equals(field.getName())) {
```

Reply:
If we add the null check to `name` above, we can avoid the null check to `field.getName()`.